evm/utils.js

```diff
@@ -1057,7 +1057,7 @@ const verifyContractByName = (env, chain, name, contract, args, options = {}) =>
 const isConsensusChain = (chain) => chain.contracts.AxelarGateway?.connectionType !== 'amplifier';
 
 /**
- * Determines if a chain is a Hyperliquid chain based on its name or axelarId
+ * Determines if a chain is a Hyperliquid chain based on its axelarId
  * @param {Object} chain - The chain object containing name and axelarId properties
```

Comment:
```suggestion
 * @param {Object} chain - The chain object containing axelarId property
```
Outdated. I'd rather delete the whole docstring. The function name is obvious. Comments add maintenance overhead, so we should only add them if we can't make the code self-descriptive